setup.py

```diff
@@ -14,7 +14,7 @@
 
 from setuptools import setup
 
-version = '1.0.1'
+version = '1.1.0'
```

Comment:
My reason is consistency. pydatalab is part of datalab so it is easier to use same version to avoid confusions (users don't care about pydatalab/datalab releases). I don't see a case that we have enough reasons to release a new pydatalab, but not datalab image. Since we are now releasing them in the same cadence, it is a good opportunity to align the versions.